go/internal/feast/registry/http.go

```diff
@@ -0,0 +1,212 @@
+package registry
```

Comment:
This needs to be switched to remote registry. We built http registry to overcome the password sharing for SQL Registry. HTTP Registry which is FastAPI based is not contributed to Feast. We are also planning to leverage Remote registry and move over from HTTP one.

Reply:
Thank you for the info, @EXPEBdodla ! Let me see try to resolve this.

Reply:
I choose an "easy" fix for this.